### ISSUE TITLE
Render tabs in tabs overview screen as WebViews instead of bitmaps

### ISSUE THREAD

**Issue by abdulwd** [kiwix/kiwix-android#1411]
**Is your feature request related to a problem? Please describe.**

Currently, in the tabs overview screen, the tabs are rendered as bitmaps but it has some shortcomings like the preview of the tab is not shown if the WebView is not loaded yet, bitmaps take up memory, etc.

**Describe the solution you'd like**

Use WebViews in TabsAdapter instead of converting them into bitmaps. The WebViews have to be resized when the tabs overview screen is displayed and again after it is hidden. The clicks on the WebViews have to be captured when the tabs overscreen is shown because for selecting a tab the user will click on the tab and that click will be propagated to the WebView which may have a href that will result in navigation to a new page even when the tabs overview screen is shown.

Should we show a single tab at a time in the overview screen or should we show tabs as tiles as done in firefox? It is also a point of discussion.

**Describe alternatives you've considered**

I found this library https://github.com/michael-rapp/ChromeLikeTabSwitcher but this will require a lot of refactoring.

**Additional context**

This is the tab overview screen (specifying it as some may not have context):
![image](https://user-images.githubusercontent.com/20206262/63949945-c62a2000-ca98-11e9-8be2-d7f985dabccd.png)

**Comment by kelson42** [comment]
@abdulwd @macgills I'm not really sure to understand properly the user impact of this bug. Do you mean the tabs preview does not always match the reality of the open tabs?

**Comment by abdulwd** [comment]
Yes, in the sense that if the WebViews in the list have not been rendered yet then there will just be a white tab in its preview.

**Comment by kelson42** [comment]
This a serious bug then.

**Comment by mhutti1** [comment]
Well I mean one could argue that its representative of the real state of the webview as I imagine they don't load while not visible.

**Comment by kelson42** [comment]
@mhutti1 Yes, you probably could... but pretty sure you shouldn't ;)

**Comment by kelson42** [comment]
The very first user feedback about the new 3.0 is a complain about that bug.

> Dear developers! Fitst, thank you so much for greates offline wikipedia app. I'm using it for 4 years with pleasure. Howewrer, last update is a bit contrversial... It is great that app is developing through the time but some UI features were better in old version. Navigation between tabs is easier in old version - you can see all open tabs (not only 3). Preview of open tabs is not that important - wikipedia tabs are mostly text... And the tab switch button is on the top now.It hard to reach it!

**Comment by abdulwd** [comment]
Then I think we should do it as Firefox does. And move the tab switcher to the bottom toolbar.

**Comment by abdulwd** [comment]
https://user-media-prod-cdn.itsre-sumo.mozilla.net/uploads/gallery/images/2017-04-18-20-07-21-0f073e.png

Check this out. Should we do something like this?

**Comment by abdulwd** [comment]
Ignore the toolbar just look at the tabs preview we can have the icon to switch tabs in the bottom toolbar. @macgills What do you suggest?

**Comment by macgills** [comment]
I know understand a lot more about web content rendering. This is infeasible until we support multizim, essentially a map of file/path/uri to a ZimFileReader so we can select from the ContentProvider the appropriate data

**Comment by kelson42** [comment]
@macgills Moving to 3.3
